internal/config/config.go

```diff
@@ -52,7 +54,7 @@ type BaseOperatorConf struct {
 	VMAlertDefault            struct {
 		Image               string `default:"victoriametrics/vmalert"`
 		Version             string `default:"v1.91.0"`
-		Port                string `default:"8080"`
+		Port                string `default:"8880"`
```

Comment:
I think, it's better to keep default port `8080` as is. And fix `cr *VMAlert) AsURL()`.

 This change may break configuration for some users, that configured `vmalert.proxyURL` param.  For instance `k8s-stack` uses it.

Reply:
Then we need to update the default port[8880->8080] vmalert listening by default in [VictoriaMetrics/VictoriaMetrics ](https://github.com/search?q=repo%3AVictoriaMetrics%2FVictoriaMetrics+8880&type=code), right?

Reply:
No, it could be different ports. I believe, we could sync it with `vmalert` defaults values before `v1.0.0` operator release.

 The same happened to `VMSingle`. Operator configures it to listen `:8429` port, but default port is `:8428`.

Reply:
Got it, fixed :)